docs/components/table-view.md

```diff
@@ -622,6 +664,135 @@ A `error-action-click` event is emitted when error state action button is clicke
 />
 ```
 
+## Expandable Rows
+
+Data presented in a table often requires futher clarification or specification. The [`expandableRows` prop](#expandablerows) allows each row to be toggled, revealing or hiding additional information without the need to navigate away from the current view. Any content can be passed using the [`row-expanded` slot](#row-expanded). However, when displaying a nested table in expanded rows, there are a few important considerations to be aware of.
+
+### Nested Table With Different Columns
+
+A nested table represents data that differs from the parent table but is loosely related. This type of nested table does not require any special handling other than setting the [`nested` prop](#nested) to `true`.
+
+Notice that column visibility, column resizing and bulk actions features, as well as `toolbar` slot, are disabled in nested tables.
+
+<KTableView
+  :headers="teamsHeaders"
+  :data="teamsData"
+  expandable-rows
+  resize-columns
+  :pagination-attributes="{ totalCount: teamsData.length }"
+>
+  <template #row-expanded>
+    <KTableView
+      :headers="basicHeaders()"
+      :data="basicData"
+      nested
+      :pagination-attributes="{ totalCount: basicData.length }"
+    />
+  </template>
+</KTableView>
+
+```html
+<KTableView
+  :headers="parentHeaders"
+  :data="parentData"
+  expandable-rows
+  resize-columns
+  :pagination-attributes="{ totalCount: parentData.length }"
+>
+  <template #row-expanded>
+    <KTableView
+      :headers="nestedHeaders"
+      :data="nestedData"
+      nested
+      :pagination-attributes="{ totalCount: nestedData.length }"
+    />
+  </template>
+</KTableView>
+```
+
+:::tip NOTE
+Both parent and nested tables in this example contain static data for demonstration purposes.
+:::
+
+### Nested Table With Same Columns
+  
+A nested table that complements the parent table requires some special handling:
+
+* The `nested` prop must be set to `true`
+* Table headers can be hidden using the [`hideHeader` prop](#hideheader). Value still need to be passed through the `headers` prop, and this is where the `nestedHeaders` slot prop becomes useful. It returns an array of header objects, simplifying the synchronization of column visibility when it is enabled in the parent table
+* To better align the columns of the parent and nested tables, the `columnWidths` slot prop can be utilized. It returns an object that can be passed to the nested table through the [`tablePreferences` prop](#tablepreferences). Each time a column is resized in the parent table, the nested table will be updated accordingly
+
+:::warning NOTE
+If bulk actions is enabled in parent table, it will be disabled in the nested table.
```

Comment:
question: why this restriction? You may want to act on the parent and children

Reply:
Nested table doesn't have room for bulk actions dropdown since toolbar is hidden. Act on both parent and nested from the parent table dropdown is unrealistic in majority of use cases since parent and nested items are usually different entities (debatable but see my next point). For not this limitation is coming purely from the fact that nested table doesn't display toolbar. Maybe in future we revisit this when we have a use case.